packages/zowe-explorer/src/uss/init.ts

```diff
@@ -191,5 +191,5 @@ export async function initUSSProvider(context: vscode.ExtensionContext): Promise
     );
 
     initSubscribers(context, ussFileProvider);
-    return ussFileProvider;
+    return ussFileProvider as USSTree;
```

Comment:
We can avoid this type cast by removing the type annotation or changing the annotation to `USSTree` on line 27.